datafusion/sqllogictest/test_files/subquery.slt

```diff
@@ -389,19 +389,25 @@ where o_orderstatus in (
 2
 3
 
+# uncorrelated exists
+query I
+SELECT 1 WHERE EXISTS (SELECT 1)
+----
+1
+
 #exists_subquery_with_same_table
 #Subquery and outer query refer to the same table.
 #It will not be rewritten to join because it is not a correlated subquery.
```

Comment:
Is this comment out of date? I guess we expect to rewrite it to a semi-join now (?

Reply:
thanks, removing!